### PR TITLE
[PB-1080]: fix/folder name on task logger

### PR DESCRIPTION
**Pull request by xabg2**
**PROBLEM:**
When the user moves a folder, the TaskLogger displays the name + ext (for example: folder.folder).

**FIX:**
Instead passing the whole object to get the name, we only pass the name to avoid mixing the name + extension.

### CODE REVIEW

File: src/app/drive/components/DriveExplorer/DriveExplorer.tsx [hunk 1]
```diff
@@ -665,7 +671,7 @@ const DriveExplorer = (props: DriveExplorerProps): JSX.Element => {
       <EditFolderNameDialog />
       <UploadItemsFailsDialog />
       <MenuItemToGetSize />
-      <BannerWrapper />
+      {envService.isProduction() && <BannerWrapper />}
```
Comment (CandelR) [inline src/app/drive/components/DriveExplorer/DriveExplorer.tsx]: Why display the banner only in production?

Comment (xabg2) [reply src/app/drive/components/DriveExplorer/DriveExplorer.tsx]: Because for testing or development it bothers, at least for me. But I can add it again

**Review by CandelR — COMMENTED** [review]
Review the code smells that sonar cloud is pointing please 👀

**Review by CandelR — APPROVED** [review]

**Review by CandelR — COMMENTED** [review]
Approved but not merge in order to wait the QA

**Pull request merged**